ONSdigital/eq-author-app
# 1076 table redesign

### What is the context of this PR?
Change the dashboard layout following trello card <https://trello.com/c/2CH7r9kp/1076-table-redesign-4>.  

Used https://github.com/ONSdigital/eq-author-app/tree/prototype-sharing as a guide.

Layout changes
* Added questionnaire short title to table. This should appear above the title in the row.
* Changed "Created by" column to "Owner".
* Made entire row clickable.
* Enabled hover styles on rows.
* Move questionnaire title to header
* Renamed "Create" to "Create questionnaire" and moved button to the right. 

### How to review 
* Check that the dashboard layout has changed according to the above changes.
* See that the animation when deleting questionnaires is working smoothly, even when adding several questionnaires.
* See that focus is working correctly after deleting a questionnaire.
* Check that the entire row is clickable.
* Use keyboard to navigate through layout.
